arbor/fvm_layout.cpp

```diff
@@ -838,6 +838,16 @@ fvm_mechanism_data fvm_build_mechanism_data(const cable_cell_global_properties&
         }
     };
 
+    // Mechanism kind as string, for error messages.
+    auto mech_kind_to_string = [](arb_mechanism_kind t) {
```

Comment:
Just an option for compression: This is essentially the same as in the Python wrapper.

Reply:
Yes, it just didn't seem "important" enough to make it into a public header file like `mechinfo.hpp`. Do you have any idea on where it should go?

Reply:
Usually I would put it next to the enum it converts, but that's in `mechanism_abi.h`. Options I see here
- make it a C function in the ABI header (I don't love this, but maybe useful)
- put it in `mechinfo.hpp` as a free function (:/ goes into the global namespace)
- put it as a member into `mechinfo` (also not in love with this)
- have it in `util/abi_helpers.hpp` or similar.
I tend to either option 1. or 4.

Reply:
Since it's not a huge duplication, and it's in separate parts of the code (core lib/python bindings), and there's no clear option on where it should go, should we keep the duplication for now? If you still think it's better to take it out, I'll make a `util/abi_helpers.hpp`